website/docs/language/functions/timeadd.mdx

```diff
@@ -1,38 +1,67 @@
 ---
-page_title: timeadd - Functions - Configuration Language
+page_title: timeadd function reference - Functions - Configuration Language
 description: |-
   The timeadd function adds a duration to a timestamp, returning a new
   timestamp.
 ---
 
-# `timeadd` Function
+# `timeadd` function reference
 
+This topic provices reference information about the `timeadd` function. 
 `timeadd` adds a duration to a timestamp, returning a new timestamp.
 
+## Introduction
+
+The Terraform language represents timestamps as strings using [RFC
+3339][rfc3339]'s Date and Time format.
+`timeadd` requires that the `timestamp` argument is a string conforming to this syntax.
+
+## Syntax
+
+Use the `timeadd` function with the following syntax:
+
+
 ```hcl
 timeadd(timestamp, duration)
 ```
 
-In the Terraform language, timestamps are conventionally represented as
-strings using [RFC 3339](https://tools.ietf.org/html/rfc3339)
-"Date and Time format" syntax. `timeadd` requires the `timestamp` argument
-to be a string conforming to this syntax.
+- `timestamp` is a string representation of a date in RFC 3339 format. See the
+  external RFC 3339's [Internet Date/Time Format section][date-time-format] for how to construct a timestamp string.
+- `duration` is a string representation of a time difference. This string consists of
+sequences of number and unit pairs, such as `"1.5h"` or `"1h30m"`. You may use
+the following units:
+
+  - `ns`: nanosecond
+  - `us` or `µs`: microsecond
+  - `ms`: millisecond
+  - `s`: second
+  - `m`: minute
+  - `h`: hour
 
-`duration` is a string representation of a time difference, consisting of
-sequences of number and unit pairs, like `"1.5h"` or `"1h30m"`. The accepted
-units are `"ns"`, `"us"` (or `"µs"`), `"ms"`, `"s"`, `"m"`, and `"h"`. The first
-number may be negative to indicate a negative duration, like `"-2h5m"`.
+  To indicate a negative duration, make the first number negative, such as "-2h5m"`.
```

Comment:
I think you may want this outside the list?
```suggestion

To indicate a negative duration, make the first number negative, such as `"-2h5m"`.
```

Reply:
Actually, nope I didn't because that sentence specifically refers to `duration`.